src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -192,9 +192,9 @@ private void generateStatusMessage(List<String> versionToQuery,
         }
 
         // the "Search More" button to continue searching from the previous fromDate
-        status.append("<button class=\"btn-link\" id=\"button_older\" onclick=\"submitFormAjax("
+        status.append("<button class=\"btn-link\" id=\"button_older\" data-next-end-time=\""
```

Comment:
Next time, try to separate these changes from the rest whenever possible, it is a great change but now if we ever revert one of them, the other one goes along with it :(